src/navbar/navbar-action-buttons.tsx

```diff
@@ -176,17 +185,43 @@ export const NavbarActionButtons = ({
 
             if (component) {
                 return (
-                    <ButtonItem key={`action-button-${index + 1}`}>
+                    <ButtonItem
+                        key={`action-button-${index + 1}`}
+                        $mobile={isMobile}
+                    >
                         {component}
                     </ButtonItem>
                 );
             }
         });
     };
 
-    if (actionButtons && actionButtons.length > 0) {
-        const ContentWrapper = mobile ? MobileWrapper : Wrapper;
-        return <ContentWrapper>{renderButtons(mobile)}</ContentWrapper>;
+    if (actionButtons && actionButtons.desktop.length > 0) {
```

Comment:
can move this check for `actionButtons.desktop.length > 0` further down to conditionally render `Wrapper`

similarly, can check if `uncollapsableActionButtons.length > 0` before rendering `MobileWrapper`

Reply:
still need the check for `actionButtons` as it's an optional prop